src/main/java/org/neo4j/gis/spatial/osm/OSMImporter.java

```diff
@@ -2131,6 +2132,15 @@ public static RoadDirection isOneway( Map<String, Object> wayProperties )
         }
         return RoadDirection.BOTH;
     }
+    
+    /**
+     * Legacy version of {@link #getRoadDirection(Map)}, for code using it under the old name.
+     * @param wayProperties the property map of the road
+     * @return same as {@link #getRoadDirection(Map)}
+     */
+    public static RoadDirection isOneway(Map<String, Object> wayProperties) {
+    	return getRoadDirection(wayProperties);
+    }
 
```

Comment:
I'm not sure if `isOneway(...)` should perhaps be marked as deprecated, but I think it should not.